be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -318,11 +318,11 @@ void HdfsOrcScanner::update_counter() {
     HdfsScanner::update_counter();
 
 #ifndef BE_TEST
-    COUNTER_UPDATE(_scanner_params.parent->_raw_rows_counter, _stats.raw_rows_read);
+    COUNTER_UPDATE(_scanner_params.parent->_rows_read_counter, _stats.raw_rows_read);
```

Comment:
Maybe we should change this for `HdfsParquetScanner::update_counter` too

Reply:
it think this has been updated in be/src/exec/vectorized/hdfs_scanner.cpp

> Maybe we should change this for `HdfsParquetScanner::update_counter` too

Reply:
Yes, already updated.